docs/development/submitting-changes.rst

```diff
@@ -0,0 +1,84 @@
+Submitting Changes
+==================
+
+This page details how to make and submit changes to the `j5` codebase.
+
+Repositories
+------------
+
+The main repository for `j5` is available on GitHub_, and Pull Requests should be submitted there.
+
+There is an additional repository available on GitLab_, although this is a mirror of the GitHub repo, and changes
+should not be submitted there.
+
+.. _GitHub: https://github.com/j5api/j5
+.. _GitLab: https://gitlab.com/j5api/j5
+
+Publishing Changes
+------------------
+
+The first step to submit changes is to publish them so that other contributors can review and give feedback.
```

Comment:
Should we encourage that substantial changes are discussed in an issue prior to a PR being raised? It could save someone a lot of effort if we turn around and say we don't really want the feature they implemented.

Reply:
Yeah, definitely. Most major changes already are, so we should get that written down.

Reply:
0890331